src/dmd/parse.d

```diff
@@ -3497,6 +3498,28 @@ final class Parser(AST) : Lexer
             t = parseVector();
             break;
 
+        case TOK.traits:
+        {
+            AST.TraitsExp te = cast(AST.TraitsExp) parsePrimaryExp();
+            if (!te)
+            {
+                // error already emitted while parsing primary
+                t = new AST.TypeError;
+            }
+            else if (strcmp(te.ident.toChars, "getMember") != 0)
```

Comment:
strcmp must die. Really, everyone, please stop using it! Use `if (te.ident == Id.getMember)`.

Reply:
Would be good to approve https://github.com/dlang/dmd/pull/7893 then, s.t. we can start using DStrings everywhere.